### PR TITLE
[v2.9] update qase parser

### PR DESCRIPTION
**Pull request by igomez06**
## Problem
 Upgrade the Qase reporting to improve readability 
 
## Solution
 Updated the Qase parser to split test cases to be under their test suite.
 
## Testing
 Ran shortened version of recurring runs.

### Automated Testing

* Test types added/modified:
    * Unit
    * Integration (Go Framework)
    * Integration (v2prov Framework)
    * Validation (Go Framework)
    * Other - Explain: _EXPLAIN_
    * None
    * _REMOVE NOT APPLICABLE BULLET POINTS ABOVE_
* If "None" - Reason: _EXPLAIN THE REASON_
  
* If "None" - GH Issue/PR: _LINK TO GH ISSUE/PR TO ADD TESTS_

Summary: _TODO_

### Regressions Considerations

_TODO_

Existing / newly added automated tests that provide evidence there are no regressions:
* _TODO_

### CODE REVIEW

File: tests/v2/validation/provisioning/k3s/provisioning_node_driver_test.go [hunk 1]
```diff
@@ -110,7 +110,7 @@ func (k *K3SNodeDriverProvisioningTestSuite) TestProvisioningK3SClusterDynamicIn
 		client *rancher.Client
 	}{
 		{provisioninginput.AdminClientName.String(), k.client},
-		{provisioninginput.StandardClientName.String(), k.standardUserClient},
+		// {provisioninginput.StandardClientName.String(), k.standardUserClient},
```
Comment (caliskanugur) [inline tests/v2/validation/provisioning/k3s/provisioning_node_driver_test.go:113]: I think this one is commented cause sanity checks

**Review by caliskanugur — COMMENTED** [review]
Looks good, just have one nit

File: tests/v2/validation/provisioning/k3s/provisioning_node_driver_test.go (hunk 1 above)
Comment (igomez06) [reply tests/v2/validation/provisioning/k3s/provisioning_node_driver_test.go:113]: ah yes, let me fix it. Thanks

**Review by caliskanugur — APPROVED** [review]
Looks good to me

**Comment by igomez06** [comment]
Backports: 

- https://github.com/rancher/rancher/pull/44772
- https://github.com/rancher/rancher/pull/44773

File: tests/v2/validation/pipeline/qase/reporter/main.go [hunk 2]
```diff
@@ -18,7 +18,7 @@ import (
 )
 
 const (
-	automationSuiteID    = int32(14)
+	automationSuiteID    = int32(554)
```
Comment (floatingman) [inline tests/v2/validation/pipeline/qase/reporter/main.go:21]: I'm curious about this number. Is there any significance to it? Will it ever need to be changed?

**Review by floatingman — COMMENTED** [review]
Just one question.

**Review by slickwarren — COMMENTED** [review]
one nit; can you update the readme, maybe this one: https://github.com/rancher/rancher/tree/release/v2.9/tests

to indicate no test name should include `/` ever? everything else lgtm

**Comment by igomez06** [comment]
> one nit; can you update the readme, maybe this one: https://github.com/rancher/rancher/tree/release/v2.9/tests
> 
> to indicate no test name should include `/` ever? everything else lgtm

Yes very good point!

File: tests/v2/validation/pipeline/qase/reporter/main.go (hunk 2 above)
Comment (floatingman) [reply tests/v2/validation/pipeline/qase/reporter/main.go:21]: Makes sense

**Review by floatingman — APPROVED** [review]

**Pull request merged**